README.md

```diff
@@ -18,13 +18,18 @@ To add a new ERC20 token you need to follow these steps:
+| ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `address`           | The lower-case address of the ERC20 token contract.                                                                                                                                                                                                          |
+| `decimals`          | The number of decimals used by the ERC20 token.                                                                                                                                                                                                              |
+| `imageUrl`          | The logo to display for the ERC20 token. It must be a 256 x 256 PNG. Add the image to the [assets/tokens](assets/tokens) folder and use the following format: `https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/{image}.png` |
+| `name`              | The name to display for the ERC20 token.                                                                                                                                                                                                                     |
+| `symbol`            | The short symbol to display for the ERC20 token.                                                                                                                                                                                                             |
+| `isStableCoin`      | Whether the token is a stable coin and the price is not expected to change                                                                                                                                                                                   |
+| `isZeroState`       | Whether the token is displayed even if the balance is zero                                                                                                                                                                                                   |
+| `isCashInEligible`  | Whether the token is eligible for cash-ins                                                                                                                                                                                                                   |
+| `isCashOutEligible` | Whether the token is eligible for cash-outs                                                                                                                                                                                                                  |
```

Comment:
Do we expect outside contributor to fill these? 
The documented fields in the README were only for the required fields to keep external contributions as simple as possible.

Reply:
Nope, these are internal only. I've removed these and included some of these in types.ts where the name isn't completely obvious.